docs/guides/developer/mkdocs.yml

```diff
@@ -27,79 +27,89 @@ extra_javascript:
-   - Security Issues: 'security.md'
-   - Code Style: 'code-style.md'
-   - Documentation: 'documentation.md'
-- Decision Making:
-   - Overview: 'decision-making/index.md'
-   - Lazy Consensus: 'decision-making/lazy-consensus.md'
-   - Consensus Building: 'decision-making/consensus-building.md'
-   - Voting: 'decision-making/voting.md'
-- Development Environment:
-   - Overview: 'development-environment.md'
-   - With Docker: 'development-environment-docker.md'
-- Committers: 'committer.md'
+- Participate:
+   - Development process: 'participate/development-process.md'
+   - Committers: 'participate/committer.md'
+   - Release manager: 'participate/release-manager.md'
+   - QA coordinator: 'participate/qa-coordinator.md'
+   - Translators: 'participate/translators.md'
+   - Submit issues/PRs: 
+      - Overview: 'participate/submit-issues-prs.md'
+      - Security issues: participate/security.md
+   - Decision making:
+      - Overview: 'participate/decision-making/index.md'
+      - Lazy consensus: 'participate/decision-making/lazy-consensus.md'
+      - Consensus building: 'participate/decision-making/consensus-building.md'
+      - Voting: 'participate/decision-making/voting.md'
+   - Governance: 'participate/governance.md'
+   - Services infrastructure:
+      - Overview: 'participate/infrastructure/index.md'
+      - Maven repository: 'participate/infrastructure/maven-repository.md'
+      - Packaging: 'participate/infrastructure/packaging.md'
+   - Proposal log: 'participate/proposal-log.md'
+- Develop:
+   - Setup dev environment:
+      - General: 'develop/setup-opencast-for-develop.md'
+      - With Docker: 'develop/development-environment-docker.md'
+   - Tips for developing: 'develop/development-environment.md'
+   - Dev ops:
+      - Local cluster for testing: 'develop/devops/local-cluster.md'
+      - Explore H2 database: 'develop/devops/explore-h2-database.md'
+   - Code style: 'develop/code-style.md'
+   - Debugging: 'develop/debugging.md'
+   - Frontend development:
+      - Overview: 'develop/admin-ui/development.md'
+      - Style:
+         - Alerts indicators: 'develop/admin-ui/style/alerts-indicators.md'
+         - Buttons: 'develop/admin-ui/style/buttons.md'
+         - Color-palette: 'develop/admin-ui/style/color-palette.md'
+         - dropdowns: 'develop/admin-ui/style/dropdowns.md'
+         - Forms: 'develop/admin-ui/style/forms.md'
+         - Icons: 'develop/admin-ui/style/icons.md'
+         - Index: 'develop/admin-ui/style/index.md'
+         - Modals: 'develop/admin-ui/style/modals.md'
+         - Navigation: 'develop/admin-ui/style/navigation.md'
+         - References: 'develop/admin-ui/style/references.md'
+         - Spacing: 'develop/admin-ui/style/spacing.md'
+         - Typeface: 'develop/admin-ui/style/typeface.md'
+         - View Structure: 'develop/admin-ui/style/view-structure.md'
+   - Documentation: 'develop/documentation.md'
+   - Localization: 'develop/localization.md'
+- Opencast architecture:
```

Comment:
We should have documentation on the workspace or even more generally on the filesystem layout

Reply:
Done! I added the `filesystem` page. 

Thanks for the suggestion!